### ISSUE TITLE
Fonts and duplicated glyph 

### ISSUE THREAD

**Issue by naveed-ahmad** [quran/quran.com-frontend#569]
tl;dr:  Its lot of work feel free to close issue :) 

#### But seriously we can drastically reduce font size or I'm really missing something. 

p1 font has 555 glyphs, actual glyph that we need are 29( go ahead and count number of words in first surah - end of ayah glyph ) or see screenshot

<img width="1234" alt="glyph" src="https://cloud.githubusercontent.com/assets/701567/21806245/00147eb2-d75a-11e6-964d-e7d82daf196e.png">

Other glyphs are ayah marks, harakts,  and stuff: 

<img width="1183" alt="stuff" src="https://cloud.githubusercontent.com/assets/701567/21806450/2f2d06f0-d75b-11e6-9a84-eaae3f940524.png">

If we are ready for challenge, I believe we can reduce size of these fonts files from 90+mb to 20-30 mb by moving ayah number and harkat to common font file. Each font file should have actual word glyph nothing more. 

BTW we can also embed tajwed colors into ttf 
* https://color.typekit.com/
* https://css-tricks.com/color-fonts/

**Comment by mmahalwy** [comment]
CC @ahmedre since he is the font guru

**Comment by ahmedre** [comment]
hardly a guru, so i'll preface with "i don't know" - i did a small experiment where i used FontForge to combine some of the fonts, and i get a reduction of around 30-50% - but it's a very manual process.

i tried removing some glyphs, but some are mapped, meaning that if you remove a set of glyphs, another set also removes by nature of removing the first ones. i am not sure why that is and if there's a way to just remove the numeric ones, but if so, i'd love any wins we can get on the font size, so am open to any suggestions on this.

**Comment by naveed-ahmad** [comment]
Well well well! I think this could works. Just did a quick test, copied 29 glyph of first page converted them into fonts and here is test for first verse: 

http://staging.quran.com:33129/1
NOTE: Bismillah before each verse is rendered using optimized fonts ( 29 glyphs only ) we can render whole page with minor changes into api( it should return unicode instead of html encoded code) 

Ill say again lets do this!

**Comment by naveed-ahmad** [comment]
We'll need to update backend as well. This will goes live with next production build inshAllah.
